draft-ietf-httpbis-digest-headers.md

```diff
@@ -559,20 +529,22 @@ metadata are discussed in {{usage-in-signatures}}.
 Digital signatures are widely used together with checksums to provide the
 certain identification of the origin of a message [NIST800-32]. Such signatures
 can protect one or more HTTP fields and there are additional considerations when
-digest fields are included in this set.
+`Representation-Digest` or `Content-Digest` is included in this set.
 
-Since digest fields are checksums of resource representations, they explicitly
+Checksums explicitly
 depend on the "representation metadata" (e.g. the values of `Content-Type`,
-`Content-Encoding` etc). A signature that protects `Digest` but not other
+`Content-Encoding` etc). A signature that protects `Representation-Digest` but not other
 "representation metadata" can expose the communication to tampering. For
 example, an actor could manipulate the `Content-Type` field-value and cause a
 digest validation failure at the recipient, preventing the application from
 accessing the representation. Such an attack consumes the resources of both
 endpoints. See also {{digest-and-content-location}}.
 
-Since signatures are meant to protect against adversarial use cases,
-insecure and collision-prone algorithms do not provide adequate guarantees
-(see {{algorithms}}).
+Digest fields SHOULD always be used over a connection that provides integrity at
+the transport layer that protects HTTP fields.
```

Comment:
```suggestion
the transport layer to protect HTTP fields from modification.
```

Separately, this could be read to imply that a TCP checksum is sufficient.  I'm not enthusiastic about that, but...ok.

Reply:
Your comment applies to the current version of the document too. we were kind of trying to lump TCP and TLS (transport layer security) together here. Maybe we can rephrase things but I would like to track that independendently of the proposal wholesale.

Reply:
tracked in #1940